opennms-dao/src/main/java/org/opennms/netmgt/dao/hibernate/DistPollerDaoHibernate.java

```diff
@@ -42,7 +42,8 @@ public DistPollerDaoHibernate() {
 
     @Override
     public OnmsDistPoller whoami() {
-        // Return the OnmsDistPoller with the default UUID
-        return get(DEFAULT_DIST_POLLER_ID);
+        // Return the OnmsDistPoller
+        final String hql = "from OnmsMonitoringSystem where type='OpenNMS'";
```

Comment:
This makes me nervous when we're not really guaranteeing one entry, but I don't suppose we're going to suddenly implement clustering of more than one "OpenNMS" system sharing a distpoller database anytime soon. :)